### ISSUE TITLE
Pipewire issues

### ISSUE THREAD

**Issue by amadejkastelic** [lawl/NoiseTorch#133]
Noise cancellation does not seem to work using Pipewire for me. I'm using the latest version of Pipewire (0.3.28) and NoiseTorch (0.11.1). The module does seem to load but it doesn't create any new input sinks. I'm using Arch Linux + KDE Plasma.

![image](https://user-images.githubusercontent.com/26391003/118934782-073c2500-b94b-11eb-9062-09c9e2cee9e2.png)

![image](https://user-images.githubusercontent.com/26391003/118934846-1d49e580-b94b-11eb-999f-45aa51354e3a.png)

**Comment by amadejkastelic** [comment]
This is the error I get when I try to load it with `noisetorch -log`:
```
2021/05/20 09:12:07 Loading supressor for pipewire
2021/05/20 09:12:07 Error loading input: PulseAudio error: commandLoadModule -> No such entity
2021/05/20 09:12:07 PulseAudio error: commandLoadModule -> No such entity
```

**Comment by amadejkastelic** [comment]
Nice! Installing from AUR works for me:
```
paru -S noisetorch
```

**Comment by lawl** [comment]
I think I've found the issue, anyone willing to test this if this dev build does indeed fix the problem?

[noisetorch.zip](https://github.com/lawl/NoiseTorch/files/6515550/noisetorch.zip)

**Comment by lorddickfist** [comment]
> I think I've found the issue, anyone willing to test this if this dev build does indeed fix the problem? (Just chmod +x and run it)
> 
> [noisetorch.zip](https://github.com/lawl/NoiseTorch/files/6515550/noisetorch.zip)

this build works for me

**Comment by lawl** [comment]
Thanks everyone, I've pushed this fix out as [0.11.2](https://github.com/lawl/NoiseTorch/releases/tag/0.11.2).
